src/main/java/com/microsoft/aad/msal4j/AuthenticationErrorCode.java

```diff
@@ -23,20 +23,10 @@
 
 package com.microsoft.aad.msal4j;
 
-public enum AuthenticationErrorCode {
+public class AuthenticationErrorCode {
 
-    UNKNOWN ("unknown"),
-    AUTHORIZATION_PENDING ("authorization_pending"),
-    INTERACTION_REQUIRED ("interaction_required");
-
-    private String errorCode;
-
-    AuthenticationErrorCode(String errorCode){
-        this.errorCode = errorCode;
-    }
-
-    @Override
-    public String toString(){
-        return errorCode;
-    }
+    public final static String AUTHORIZATION_PENDING = "authorization_pending";
```

Comment:
Consider providing comments on these strings, in a format similar to "What Happens?" and "Mitigation".

Reply:
imo, a must as these are public consts. Please document the usages, meaning and mitigations.

---
In reply to: [300341764](https://github.com/AzureAD/microsoft-authentication-library-for-java/pull/69#discussion_r300341764) [](ancestors = 300341764)